sway-lib-std/src/u256.sw

```diff
@@ -88,4 +91,286 @@ impl U256 {
     pub fn bits() -> u32 {
         256
     }
+
+    /// Get 4 64 bit words from a single U256 value.
+    fn decompose(self) -> (u64, u64, u64, u64) {
+        (self.a, self.b, self.c, self.d)
+    }
+}
+
+impl core::ops::Ord for U256 {
+    pub fn gt(self, other: Self) -> bool {
+        if self.a > other.a {
+            return true
+        } else if self.a == other.a {
+            if self.b > other.b {
+                return true
+            } else if self.b == other.b {
+                if self.c > other.c {
+                    return true
+                } else if self.c == other.c {
+                    if self.d > other.d {
+                        return true
+                    }
+                }
+            }  
+        }
+        false
```

Comment:
OK, I know I compained about the other long line and suggested nested ifs and return statements, but this isn't great either.  My fault.  But I think this is the same and might be the simplest yet still readable version?
```suggestion
        self.a > other.a
            || (self.a == other.a && self.b > other.b
                || (self.b == other.b && self.c > other.c
                    || (self.c == other.c && self.d > other.d)))
```